source/blog/2018-08-27-ember-3-4-released.md

```diff
@@ -24,8 +24,40 @@ Ember.js is the core framework for building ambitious web applications.
 
 Ember.js 3.4 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is one (1) new feature, three (3) deprecations, and eight (8) bugfixes in this version.
 
-#### New Features (X)
+#### New Features (2)
+Angle Bracket Invocation (1 of 2)
+
+In Ember 3.4 it is now possible to invoke angle bracket components. This means that you're now able to replace the classic invocation syntax:
+
+```hbs
+{{site-header user=this.user class=(if this.user.isAdmin "admin")}}
+
+{{#super-select selected=this.user.country as |s|}}
+  {{#each this.availableCountries as |country|}}
+    {{#s.option value=country}}{{country.name}}{{/s.option}}
```

Comment:
Thoughts on making the classic syntax example match angle bracket?

`{{#super-select selected=this.user.country as |s|}}`

vs having option instead of "s"

`{{#super-select selected=this.user.country as |option|}}`

Reply:
Good catch, yeah that's probably more helpful that way. Thanks!